src/main/java/com/commercetools/project/sync/service/impl/CustomObjectServiceImpl.java

```diff
@@ -22,9 +22,9 @@
 
 public class CustomObjectServiceImpl implements CustomObjectService {
 
-  private static final String TIMESTAMP_GENERATOR_CONTAINER_POSTFIX = "timestampGenerator";
   public static final String TIMESTAMP_GENERATOR_KEY = "latestTimestamp";
-  private static final String TIMESTAMP_GENERATOR_VALUE = "";
+  public static final String TIMESTAMP_GENERATOR_CONTAINER_POSTFIX = "timestampGenerator";
+  public static final String TIMESTAMP_GENERATOR_VALUE = "";
   private static final long MINUTES_BEFORE_CURRENT_TIMESTAMP = 2;
```

Comment:
I could not get why it's 2 minutes?

Reply:
can you check the concept here by @butenkor https://docs.google.com/drawings/d/1qWcYg2Sb42TWlNHVFMj-SJ5xSINGVIVJyArWxD10dOg/edit?usp=drive_web&ouid=101363556912401994049